### DISCUSSION TITLE
I have a q about ru tools

### DISCUSSION THREAD

**Discussion by fowlmouse** [billzhong/dell-precision-3240-compact-hackintosh#2] [Q&A] [answered]
can someone help explaining how to used ru tool SOP and how to build into mackintosh install usb flash device? Thanks

**Comment by ghost** [comment, marked as answer]
https://nstarke.github.io/0037-modifying-bios-using-ru-efi.html

**Comment by Grisu70** [comment]
Hi @fowlmouse  did you try to modify settings ?
Did you manage ?

tks in advance
FAbrizio
